### ISSUE TITLE
Balance Analytics: duplicate Showcase listing

### ISSUE THREAD

**Issue by rphair** [cardano-foundation/developer-portal#1371]
@Balance-Analytics can you submit a PR to merge the duplicate listing for BALANCE currently showing here:
- https://developers.cardano.org/showcase?tags=analytics&tags=pooltool

... or post here the data that should be preserved in a merge of these two listings, and I'll submit it?  Somehow we didn't catch this duplication (perhaps until the portfolio was sorted):
- https://github.com/cardano-foundation/developer-portal/pull/946
- https://github.com/cardano-foundation/developer-portal/pull/1086

Also: this is not an issue for the Developer Portal but as far as I can tell this site only displays in bare HTML (no graphs or styles) in Firefox on Linux.  cc @ccgarant

**Comment by katomm** [comment]
Problem here is just that Balance was added two times.😅

**Comment by rphair** [comment]
yes, with those 2 times too close together 😜 ... with no feedback from @Balance-Analytics I'll submit a PR with an conjunctive merge of the array content & the image from their own original PR.

**Comment by rphair** [comment]
Fixed by https://github.com/cardano-foundation/developer-portal/pull/1374.
